docs/spec/components/schemas/Transfer.yaml

```diff
@@ -57,12 +57,12 @@ allOf:
         type: object
         required: [creator]
         properties:
-          token:
+          item:
```

Comment:
This is a breaking change. Are there any clients depending on the old version?

Reply:
only horizon https://github.com/rarimo/relayer-svc/pull/2